apps/api/src/app/user/dtos/update-profile-request.dto.ts

```diff
@@ -8,5 +8,5 @@ export class UpdateProfileRequestDto {
   lastName: string;
 
   @ApiProperty()
-  imageUrl: string;
+  profilePicture?: string;
```

Comment:
changed this to be consistent with the entity name

Reply:
@antonjoel82 yes, agree but on the other hand, it won't be aligned with the `UserResponseDto` which will return the `profilePicture`